snare.py

```diff
@@ -135,32 +135,33 @@ def create_data(self, request, response_status):
 
     async def handle_html_content(self, content):
         soup = BeautifulSoup(content, 'html.parser')
-        for p_elem in soup.find_all('p'):
-            if p_elem.findChildren():
-                continue
-            css = None
-            if 'style' in p_elem.attrs:
-                css = cssutils.parseStyle(p_elem.attrs['style'])
-            text_list = p_elem.text.split()
-            p_new = soup.new_tag('p', style=css.cssText if css else None)
-            for idx, word in enumerate(text_list):
-                # Fetch dorks if required
-                if len(self.dorks) <= 0:
-                    self.dorks = await self.get_dorks()
-                word += ' '
-                if idx % 5 == 0:
-                    a_tag = soup.new_tag(
-                        'a',
-                        href=self.dorks.pop(),
-                        style='color:{color};text-decoration:none;cursor:text;'.format(
-                            color=css.color if css and 'color' in css.keys() else '#000000'
+        if self.run_args.no_dorks != True:
```

Comment:
How about we move the whole dork insertion code into a separate function?

Reply:
I think we can add it as an issue and work on it as a separate task :)